rpcs3/util/vm_native.cpp

```diff
@@ -223,7 +223,8 @@ namespace utils
 #ifdef _WIN32
 
 		DWORD old;
-		if (!::VirtualProtect(pointer, size, +prot, &old))
+	if (::VirtualProtect(pointer, size, +prot, &old))
```

Comment:
this doesn't seem to make any difference and it's intendation is wrong... 🤔

Reply:
ALlso better squash to commits if git history is a concern.

Reply:
Kinda merged this half asleep, I've fixed it correctly.